### ISSUE TITLE
Kotlin template is only applied to method parameter instead of whole expression

### ISSUE THREAD

**Issue by exaV** [xylo/intellij-postfix-templates#163]
In Kotlin postfix templates the `$expr$` variable does not contain a valid expression if it ends with a lambda.

Template:
```
.parsedExpr: print expression
    ANY  →  println("$expr$")
```

Usage:
```kotlin

val abc = listOf("a","b","c")

// abc.parsedExpr
println("abc") // OK

// abc.map { it.toUpperCase() }.parsedExpr
abc.map println ("{ it.toUpperCase() }") // ERROR
// expected: println("abc.map { it.toUpperCase() }")


// it works if it does not end in a lambda
// abc.map { it.toUpperCase() }.toString().parsedExpr
println("abc.map { it.toUpperCase() }.toString()") // OK
```

**Comment by xylo** [comment]
Might be solved with the fix of #161.

**Comment by xylo** [comment]
Fixed in version 2.11.2.
